datameta/api/metadatasets.py

```diff
@@ -167,6 +173,24 @@ def post(request: Request) -> MetaDataSetResponse:
         submission_id = None
     )
     db.add(mdata_set)
+
+    if replaces:
+        mset_repl_evt = MsetReplacementEvent(
+            user_id = auth_user.id,
+            datetime = datetime.utcnow(),
+            label = replaces_label,
+            new_metadataset_id = mdata_set.id
+        )
+        db.add(mset_repl_evt)
+
+        for mset_id in replaces:
+
+            target_mset = resource_by_id(db, MetaDataSet, mset_id)
+            if target_mset is None:
+                raise HTTPForbidden()
```

Comment:
The invalid replacement IDs should be collected and returned as a validation error